Lib/fontTools/merge.py

```diff
@@ -1252,7 +1253,14 @@ def main(args=None):
 		args = sys.argv[1:]
 
 	options = Options()
-	args = options.parse_opts(args)
+	args = options.parse_opts(args, ignore_unknown=['output-file'])
```

Comment:
why are you parsing it outside of the Options class, does it not support plan string args? I know at some point we should get rid of that and use stock argparse, bot for now.. I guess this is ok.

Reply:
This is essentially copied from `subset/__init__.py`.

Reply:
ok